be/src/exec/pipeline/aggregate/aggregate_streaming_source_operator.cpp

```diff
@@ -47,10 +47,14 @@ void AggregateStreamingSourceOperator::set_finishing(RuntimeState* state) {
     _is_finished = true;
 }
 
+void AggregateStreamingSourceOperator::set_finished(RuntimeState* state) {
+    _aggregator->set_finished();
+}
+
 Status AggregateStreamingSourceOperator::close(RuntimeState* state) {
     // _aggregator is shared by sink operator and source operator
     // we must only close it at source operator
```

Comment:
Sink aggregator will be closed at both sink/source operators, change or remove this comment